### ISSUE TITLE
issues regarding training

### ISSUE THREAD

**Issue by sjooyoo** [galsang/CNN-sentence-classification-pytorch#1]
train 할때 test set 이 들어가면 안되지 않나요? 
코드를 보면 반영하신것같아서요..
감사합니다

**Comment by galsang** [comment]
test set이 train set에 포함된 것은 아니고
model selection에 dev set 대신 test set이 사용되었던 버그가 있어서 이를 수정하였습니다.
실험 결과 부분도 수정하였으니 확인해 보시면 됩니다.
지적 감사합니다.

There was a bug that the test set is utilized for model selection instead of dev set.
It is fixed.
